react/spring-page/server/.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST=localhost
```

Comment:
You should not commit `.env`. Rather, you should put `.env.example` to provide key names.